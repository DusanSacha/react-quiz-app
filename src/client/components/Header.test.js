import React from 'react';
import Header from './Header';
import renderer from 'react-test-renderer';

describe('Header renders correctly', () => {

  test('without content', () => {
    const tree = renderer.create(
        <Header />
    );
    expect(tree).toMatchSnapshot();
  });

  test('with header content', () => {
    const tree = renderer.create(
        <Header>React Quiz App</Header>
    );
    expect(tree).toMatchSnapshot();
  });

});