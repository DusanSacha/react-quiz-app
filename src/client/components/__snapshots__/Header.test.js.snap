// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly with header content 1`] = `
<h1
  className="sc-bdVaJa hUjkp"
>
  React Quiz Fail
</h1>
`;

exports[`Header renders correctly without content 1`] = `
<h1
  className="sc-bdVaJa hUjkp"
/>
`;
